apps/api/src/test-data/buildCategoryTestData.ts

```diff
@@ -7,7 +7,7 @@ export default (): CategorizedTree =>
     [
         {
             id: '0',
-            label: '',
+            label: 'tree of knowledge',
```

Comment:
capitalize?

Reply:
Could do, up to you.  We can always capitalize in the app.